fip-0026.md

```diff
@@ -394,14 +430,24 @@ This removes a marketplace configuration. A marketplace configuration cannot be
 |Marketplace has listings associated|Supplied `marketplace` has active listings associated with this marketplace|400| |"Marketplace has active listings."|
 |Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
 
-#### Response
-TBD 
+#### Response body
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|OK if successful|
+|fee_collected|Int|Amount of SUFs collected as fee|
 
 ##### Example
-TBD
+```
+{
+  "status": "OK",
+  "fee_collected": 2000000000
+}
+```
 
 ### Set Commission Fee
-
+#### Contract: fio.escrow
+#### New fee: set_marketplace_commission_fee
+#### Ram Increase: To be determined during implementation
 #### New action: *setmkpcomfee*
 This sets the commission fee given to the marketplace. This can only be used if there are no domains listed associated with the marketplace.
```

Comment:
the rationale behind only when there are no domains is because when I list a domain a commission is set, say 5% but a period of time can pass between when I list it and when it sells, so if 30 days go by and the commission is able to be changed then it could be 10% (hypothetically) and I am suddenly charged 10% of the sale price instead of the expected 5% when I listed it. 

Granted, changing the commission fee is likely to be rare and after some more internal discussion tonight, we decided to take out that check.

Reply:
Assuming the initiatives is successful, there will always be domains listed for sale. So if it can be changed only when no listings, it would mean it can only be changed at the time of deployment and if so you might as well just hard-coded it.

Reply:
Right, we have reconsidered that restriction. That line is removed in the latest commits.

Reply:
Please describe in FIP what happens when the fee is changed after someone lists a domain and before it's bought.

Reply:
This is updated in the fip. the fee is stored with the domain sale so that if the fee is changed, the current listings will only be charged what is in the `domainsale` table.